### PR TITLE
Updates from orb review

### PR DESCRIPTION
**Pull request by tmcneal**
Addresses a number of issues that came up during review of the orb:
- Rename the usage example.
- Use a better description for the orb.
- Remove the 'run' job and just expose a 'run_tests' command.
- Remove the unused executor.

### CODE REVIEW

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:35f50c80570bf75b30ac8fda492bdcdd2f999ead

**Review by fitzn — APPROVED** [review]

File: src/examples/run-reflect-tests.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 description: >
-  Executes a Reflect test suite.
+  Run end-to-end tests using Reflect's no-code testing platform.
 usage:
   version: 2.1
   orbs:
-    reflect: reflect/reflect-circleci-orb@0.1.0
+    reflect: reflect/reflect@0.1.0
```
Comment (KyleTryon) [inline src/examples/run-reflect-tests.yml:6]: The next release of this orb will likely be `0.2.0` or `1.0.0`, the version shown here in the description should match what the next release of the orb will be.

**Review by KyleTryon — CHANGES_REQUESTED** [review]
If the usage example is accurate, where it shows using a job, this would be the easiest way to interact with your product if there is nothing more the user needs to set up. If this is the case, I would highly recommend adding the job back to this orb.

Commands are useful when users need or want to define their own jobs and use your commands within them. If executing your tests is a simple trigger that requires nothing more from the user, a job is the fastest and easiest way to connect to your platform.

In fact, if there are no scenarios where the user should define their own job using your command, we should actually remove the command instead of the job.

What is the best option depends on the use case.

Example, https://circleci.com/developer/orbs/orb/circleci/github-cli
The GitHub CLI orb provides a _command_ to install the CLI so the user can do anything they want in a job.
It also provides a _release_ _job_ which is self contained, and can be executed directly as a job.

**Comment by tmcneal** [comment]
> If the usage example is accurate, where it shows using a job, this would be the easiest way to interact with your product if there is nothing more the user needs to set up. If this is the case, I would highly recommend adding the job back to this orb.

OK I'll restore the job. It seems the only two ways to invoke a shell script from a job is either (1) inlining it or (2) calling a command which then invokes the script defined in `command`. Assuming these are the only two options, I'm going to keep the command because I'd rather not define the script inline.

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:0cd9ebac3fa1f116758ad9d723dcf07633a80f0f

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:1b1c5a7953648af6df02361ec444ed4d2a4b3966

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:af307a591b3d0bc149d90ab0d1164e15e4f13578

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:a3d67cf7392dd94c3f66d30d6a5c22e9674fe90c

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:4b3ac1a93a2d32136752fec534be64742232fcb7

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:d27c1eb79dd6819211547bb72da1bf87db9c8d7f

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:ff7f1e3ce524c569d3123fb9bf94bebfc90c4808

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:d9d09afa779d890ec31a9a1c44b7dbae451bcbab

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:5fe4e866940a2fc6a2ed49b9a99ef2dc1e949766

File: src/examples/run-reflect-tests.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
   workflows:
     build:
       jobs:
```
[the remark this answers is not in the document]
Comment (tmcneal) [reply src/examples/run-reflect-tests.yml:9]: I've re-added the job 👍

File: src/examples/run-reflect-tests.yml (hunk 1 above)
Comment (tmcneal) [reply src/examples/run-reflect-tests.yml]: Fixed

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:d42a37fb66a49dea1533b66aff7718b5d36a4b88

**Comment by reflectpublic** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=dev:d42a37fb66a49dea1533b66aff7718b5d36a4b88

**Comment by reflectpublic** [comment]
Your orb has been published to the CircleCI Orb Registry.
You can view your published orb on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/reflect/reflect?version=0.2.0

**Pull request merged**